### ISSUE TITLE
Support for Safari

### ISSUE THREAD

**Issue by dhruvit-r** [jswanner/DontFuckWithPaste#111]
Hey @jswanner. Really appreciate the work you've done here! Just wanted to know if there are any plans to create/support a version of this app for Safari?

**Comment by jswanner** [comment]
Hey @dhruvit-r, thanks for the kind words. Great question regarding Safari; I've been using Safari more lately and so have run into occasions when I wanted to use this extension there as well. I haven't put any effort into that yet, but it is something I've thought about.

The first hurdle I would need to get over is installing Xcode, which for whatever reason is always something I avoid for as long as I can.

**Comment by dhruvit-r** [comment]
@jswanner Totally understandable. I don't have enough bandwidth at this point, but I can work on and support a fork that is specifically for Safari at some point of time in the future. Would that be something you're comfortable with?

**Comment by jswanner** [comment]
@dhruvit-r yeah, if you are willing to take that on, I'd be comfortable with that.

**Comment by cmaring** [comment]
@dhruvit-r any progress yet? I'm also looking forward to use DFWP on Safari.

**Comment by dhruvit-r** [comment]
@cmaring I created a very "crude" version that I personally use, but procrastinated on putting in the "finishing touches" and  publishing/pushing it :)

I've attached the version I use. It's probably outdated and it's not super polished, but it gets the job done (I really only have a couple websites that bug me, and this takes care of those).

[Don't Fuck With Paste.app.zip](https://github.com/jswanner/DontFuckWithPaste/files/6779968/Don.t.Fuck.With.Paste.app.zip)

**Comment by dhruvit-r** [comment]
Pushed source code here: https://github.com/dhruvit-r/DontFuckWithPaste.

**Comment by cmaring** [comment]
> Pushed source code here: https://github.com/dhruvit-r/DontFuckWithPaste.

Thanks. Just got it pulled and  installed and it does get the job done.
